### DISCUSSION TITLE
How to filter search results by sitecollection or metadata

### DISCUSSION THREAD

**Discussion by MoeIcI** [microsoft-search/pnp-modern-search#2468] [Q&A] [answered]
I'm trying to use the search as search bar for an wiki. The goal is to show only Pages which are part of the wiki sitecollection.
My plan was to filter them on one of these two options:
- tag/choice which is given to all wiki pages exclusivly
- sitecollection.

While testing the search results always show some pages or files from diffenrent site collections which is distracting.

Can anyone help me?
I couldn't find an answer in the documentation.
Thanks for your help in advance :)

Edit: I actually managed to filter the results by sitecollection.
But Still is there an oppurtunity to utilize addditional details from the page library like columns, views etc? This would make it much easier to filter by them.

**Comment by wobba** [comment, marked as answer]
Depends on your set up, and I recommend reading https://microsoft-search.github.io/pnp-modern-search/scenarios/set-up-managed-properties/ /  https://learn.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search

**Reply by MoeIcI** [reply]
Thanks for your reply!

I managed to get Tags with help of this (https://github.com/microsoft-search/pnp-modern-search/issues/610#issuecomment-744532231) and your reply.
![Unbenannt](https://user-images.githubusercontent.com/108660028/197472912-742c6eac-95c7-4570-bb7f-6bf47673191f.png)

Next step would be to filter the results to show only one MetaDataTag and skip every result without any tag in this category.
I tried to achieve this by adding a Refining filter like that:
`MetaDataTag:equals("WikiPage")`
But I couldnt get it running.
What's the best way to achieve this?

**Reply by wobba** [reply]
I would rather add KQL in the query template:

`NOT MetaDataTag=WikiPage`
